docs/docs/code/nodejs/README.md

```diff
@@ -55,7 +50,7 @@ In this example, we'll pretend this data is coming into our HTTP trigger via POS
 }
 ```
 
-In our Node.js step, we can access this data in the `steps` variable Specifically, this data from the POST request into our workflow is available in the `trigger` property. 
+In our Node.js step, we can access this data in the `steps` variable Specifically, this data from the POST request into our workflow is available in the `trigger` property.
```

Comment:
```suggestion
In our Node.js step, we can access this data in the `steps` variable. Specifically, this data from the POST request into our workflow is available in the `trigger` property.
```

Reply:
Added a period and capitalized "Specifically" to follow Google's style guide on sentence structure